### PR TITLE
Migrate from molecule to packer entirely

### PR DESCRIPTION
**Pull request by ossareh**
molecule is being deprecated and our build process is being changed to
support using infrastructure available within twitch. These changes are
made such that builds can still take place outside of twitch. To this
end there are two primary additions.

 - ./build.sh
 - ./build/packer.json

These work together to prep the environment for packer to build an AMI.
Packer now replaces much of what molecule was doing. `build.sh` handles
the pre-packer stages of the build including building the binary, which
it cross compiles for linux/amd64.

Note: the build is not responsible for running godep restore and test,
that falls on the shoulders of the person(/thing) running `./build.sh`

### CODE REVIEW

**Comment by ossareh** [comment]
I decided the presence of the "Note" on the PR actually represented a problem.. so now godep restore and test are the responsiblity of `./build.sh`

File: README.md [hunk 1]
```diff
@@ -27,6 +27,34 @@ creation of those tables.
 Systems that wish to know about the schemas currently route to scoop
 by way of blueprint.
 
+## Building
+
+```
+apt-get -u install build-essential libgeoip-dev libgeoip1`
```
Comment (aasted) [inline README.md]: I think this is a stray trailing tick

Comment (ossareh) [inline README.md]: fixed

**Comment by aasted** [comment]
This looks good to me.

**Pull request merged**